cfgov/housing_counselor/jinja2/housing_counselor/index.html

```diff
@@ -47,29 +47,57 @@ <h1>Housing counselors near you</h1>
-                            The counseling agencies on this list are
-                            approved by the U.S. Department of Housing
-                            and Urban Development (HUD) and they can
-                            offer independent advice about whether a
-                            particular set of mortgage loan terms is a
-                            good fit based on your objectives and
-                            circumstances, often at little
-                            or no cost to you. This list will show you
-                            several approved agencies in your area.
-                            There is also a
-                            <a class="a-link a-link__icon"
-                               href="https://apps.hud.gov/offices/hsg/sfh/hcc/hcs.cfm">
-                                <span class="a-link_text">list of nationwide HUD-approved counseling agencies</span>
-                                {{ svg_icon( 'external-link' ) }}</a>.
-                        </p>
+                                u-screen-only
+                                o-sidebar-breakout">
+                        <div class="content-l">
+                            <div class="o-sidebar-breakout_col
+                                        content-l_col
+                                        content-l_col-2-3">    
+                                <h1>Find a housing counselor</h1>
+                                <p>
+                                    Housing counselors throughout the country can
+                                    provide advice on buying a home, renting,
+                                    defaults, foreclosures, and credit issues.
+                                    Using the search box below,
+                                    you can find one near you.
+                                    The counseling agencies on this list are
+                                    approved by the U.S. Department of Housing
+                                    and Urban Development (HUD) and they can
+                                    offer independent advice about whether a
+                                    particular set of mortgage loan terms is a
+                                    good fit based on your objectives and
+                                    circumstances, often at little
+                                    or no cost to you. This list will show you
+                                    several approved agencies in your area.
+                                    There is also a
+                                    <a class="a-link a-link__icon"
+                                       href="https://apps.hud.gov/offices/hsg/sfh/hcc/hcs.cfm">
+                                        <span class="a-link_text">list of nationwide HUD-approved counseling agencies</span>
+                                        {{ svg_icon( 'external-link' ) }}</a>.
```

Comment:
```suggestion
                                    <a href="https://apps.hud.gov/offices/hsg/sfh/hcc/hcs.cfm">list of nationwide HUD-approved counseling agencies</a>.
```

We shouldn't need to add the external link markup manually; our Django [middleware](https://github.com/cfpb/consumerfinance.gov/blob/f7c009ab12d464771e0f371282165033f58c374f/cfgov/core/utils.py#L100-L108) should do this automatically. By relying on that middleware instead of adding manually we reduce the number of places to update in future should we want or need to change this markup.

Reply:
Thanks for catching this! I noticed it in the original code but wasn't sure if it was still needed. Removed in https://github.com/cfpb/consumerfinance.gov/pull/6249/commits/3bf444ec17339eede551a24da7e781398d6ea85b